### ISSUE TITLE
v0.3.14 error win32 x64

### ISSUE THREAD

**Issue by aimanhaizal** [microsoft/vscode-java-installer#206]
:warning:Please attach **%TEMP%\vscode-java-installer\log.txt** and **%TEMP%\vscode-java-installer\AdoptOpenJDK-MSI.log** for more details.

version: 0.3.14
env:
```
JAVA_HOME: undefined
PATH:
C:\Program Files\Google\Chrome\Application
C:\Windows\system32
C:\Windows
C:\Windows\System32\Wbem
C:\Windows\System32\WindowsPowerShell\v1.0\
C:\Windows\System32\OpenSSH\
C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common
C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR
C:\Program Files (x86)\ibm\gsk8\lib
C:\Program Files (x86)\ibm\gsk8\lib
C:\PROGRA~2\IBM\SQLLIB\BIN
C:\PROGRA~2\IBM\SQLLIB\FUNCTION
C:\PROGRA~2\IBM\SQLLIB\SAMPLES\REPL
C:\Users\Aiman\AppData\Local\Microsoft\WindowsApps
C:\Program Files (x86)\FreeArc\bin
JDKs:

```
log:
```
[12:11:26 PM] AdoptOpenJDK-11-x64
[12:12:09 PM] Download JDK ... [Completed]
[12:12:09 PM] Install JDK
[12:12:09 PM] Installing AdoptOpenJDK-11-x64...
[12:12:13 PM] Install JDK ... [Completed]
[12:12:13 PM] Download Visual Studio Code
[12:12:13 PM] arch:x64
[12:12:38 PM] Download Visual Studio Code ... [Completed]
[12:12:38 PM] Install Visual Studio Code
[12:12:48 PM] Install Visual Studio Code ... [Completed]
[12:12:48 PM] Configure Java Settings
[12:12:48 PM] Error occurred
[12:12:48 PM] Error: Java Home not found.

```

**Comment by Eskibear** [comment]
See #205 
The latest AdoptOpenJDK 11 changed its default installation directory, leading to a false alarm. You can directly install [Java Extension Pack](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), and it should work out of box.

**Comment by Eskibear** [comment]
Fixed in v0.3.15. Download link here: https://aka.ms/vscode-java-installer-win
